src/components/templates/TalkShowStudio/TalkShowStudio.tsx

```diff
@@ -194,46 +194,25 @@ export const TalkShowStudio: FC<TalkShowStudioProps> = ({ venue }) => {
     !stage.isUserOnStage &&
     !stage.isUserRequesting;
 
-  const renderScreenSharing = (
-    screenTrack: VideoPlayerProps["videoTrack"],
-    cameraTrack: VideoPlayerProps["videoTrack"]
-  ) => {
-    if (!screenTrack || !cameraTrack) return;
-    return (
+  const renderScreenSharing = (screenTrack: VideoPlayerProps["videoTrack"]) =>
+    screenTrack && (
```

Comment:
Instead of nesting everything in a conditional render like this, use our standard guard pattern:

```typescript
if (!screenTrack) return

return theJSXHere
```

Reply:
Duplicate of changes made in https://github.com/sparkletown/sparkle/pull/1653/files#r659515480